service/src/main/kotlin/no/nav/su/se/bakover/service/søknadsbehandling/SøknadsbehandlingService.kt

```diff
@@ -53,12 +56,78 @@ interface SøknadsbehandlingService {
 
     data class VilkårsvurderRequest(
         val behandlingId: UUID,
-        val behandlingsinformasjon: Behandlingsinformasjon,
-    )
+        private val behandlingsinformasjon: Behandlingsinformasjon,
+    ) {
+        fun Behandlingsinformasjon.Formue.Verdier.depositumHøyereEnnInnskudd() =
+            if (this.depositumskonto != null && this.innskudd != null) this.depositumskonto!! > this.innskudd!! else false
```

Comment:
Denne funksjonen kan godt flyttes inn i `Behandlingsinformasjon.Formue.Verdier`. Jeg hadde forventet `true` dersom this.innskudd == null && this.depositumskonto > 0

Det blir også mer naturlig og teste den der.

Reply:
det gir ikke mening at innskudd skal være null, og depositum har en verdi. Innskudd er depositum + 'annet'.